e2e/tests/functional/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -315,9 +296,45 @@ test.describe('Example Imagery in Display Layout', () => {
     await page.locator('div[title="Resize object height"] > input').click();
     await page.locator('div[title="Resize object height"] > input').fill('100');
 
-    expect(thumbsWrapperLocator.isVisible()).toBeTruthy();
-    await expect(thumbsWrapperLocator).not.toHaveClass(/is-small-thumbs/);
-  });
+        expect(thumbsWrapperLocator.isVisible()).toBeTruthy();
+        await expect(thumbsWrapperLocator).not.toHaveClass(/is-small-thumbs/);
+    });
+
+    /**
+     * Toggle layer visibility checkbox by clicking on checkbox label
+     * - should toggle checkbox and layer visibility for that image view
+     * - should NOT toggle checkbox and layer visibity for the first image view in display
+     */
+    test('Toggle layer visibility by clicking on label', async ({ page }) => {
+        await createImageryView(page);
```

Comment:
Let's add an annotation referencing the issue# at the top of this regression test:

```suggestion
        test.info().annotations.push({
          type: 'issue',
          description: 'https://github.com/nasa/openmct/issues/6709'
        });
        await createImageryView(page);
```